descriptors/ModuleDescriptor-template.json

```diff
@@ -1286,6 +1290,16 @@
       "displayName": "circulation - end patron action session",
       "description": "end patron action session"
     },
+    {
+      "permissionName": "circulation.scheduled-age-to-lost.post",
+      "displayName": "circulation - age to lost process",
+      "description": "age some borrowed items to lost"
```

Comment:
Is this permission for scheduling age-to-lost process?  If so, wouldn't a description like "schedule process for aging overdue items to lost" be better?

Reply:
Good question. 

This is a bit of a compromise. The endpoints make the scheduled aspect explicit (which I felt was important at the time of that work) and we follow a convention with our permission names to line up with the paths. However, in order to minimize the code changes (and the likelihood of mistakes), I kept the paths the same even though these are no longer scheduled by the system. That said, they are still intended to be scheduled, just not by the system.

I'm happy to change the description to make that clearer. Something like `schedule process for ageing overdue items to lost` (a minor tweak on your suggestion)?

Reply:
That sounds fine.